webapp/components/interop.html

```diff
@@ -0,0 +1,412 @@
+      display: block;
+      padding: 0.5em 0;
+      width: 100%;
+    }
+
+    a {
+      text-decoration: none;
+      color: #0d5de6;
+      font-family: monospace;
+    }
+
+    a:hover {
+      cursor: pointer;
+      color: #226ff3;
+    }
+
+    table {
+      width: 100%;
+      border-collapse: collapse;
+    }
+
+    tr.spec {
+      background-color: #eee;
+    }
+
+    tr td {
+      padding: 0 0.5em;
+    }
+
+    tr.spec td {
+      padding: 0.2em 0.5em;
+      border: solid 1px #ccc;
+    }
+
+    .path-separator {
+      padding: 0 0.1em;
+    }
+
+    .links {
+      margin-bottom: 1em;
+    }
+
+    .caveat {
+      padding: 0.5em;
+      background-color: #fde2e3;
+      margin-bottom: 1em;
+      border-left: solid 4px #f47477;
+    }
+
+    .info {
+      padding: 0.5em;
+      background-color: #e2e3fd;
+      margin-bottom: 1em;
+      margin-top: 2em;
+      border-left: solid 4px #7477f4;
+    }
+
+    .info small {
+      float: right;
+    }
+
+    @media (max-width: 800px) {
+      table tr td:first-child::after {
+        content: "";
+        display: inline-block;
+        vertical-align: top;
+        min-height: 30px;
+      }
+    }
+  </style>
+
+  <h2>Interoperability</h2>
+
+  <section class="caveat">
+    Data below are intended for web platform implementers and do not contain
+    useful metrics for evaluation or comparison of web platform features. Also
+    note that tested Edge and Safari are not pre-release versions (<a
+          href="https://github.com/w3c/wptdashboard/issues/109">#109</a>, <a
+          href="https://github.com/w3c/wptdashboard/issues/110">#110</a>).
+  </section>
+
+  <section class="runs">
+    <table>
+      <thead>
+      <tr>
+        <template is="dom-repeat" items="{{ allMetrics.metadata.test_runs }}"
+                  as="testRun">
+          <th>
+            <test-run test-run="[[testRun]]"></test-run>
+          </th>
+        </template>
+      </tr>
+      </head>
+    </table>
+  </section>
+
+  <section class="search">
+    <template is="dom-if" if="{{ !pathIsATestFile }}">
+      <input value="{{ query::input }}"
+             class="query"
+             placeholder="{{ queryPlaceholder }}"
+             data-foo="bar">
+    </template>
+    <div class="path">
+      <a href="/interop/" on-click="_navigate">WPT</a>
+      <template is="dom-repeat" items="{{ splitIntoLinkedParts(path) }}" as="part">
+        <span class="path-separator">/</span>
+        <a href="/interop{{ part.path }}"  on-click="_navigate">{{ part.name }}</a>
+      </template>
+    </div>
+  </section>
+
+  <template is="dom-if" if="{{ pathIsATestFile }}">
+    <div class="links">
+      <ul>
+        <li>
+          <a href$="https://github.com/w3c/web-platform-tests/blob/master[[path]]"
+             target="_blank">View source on GitHub</a></li>
+        <li><a href$="https://w3c-test.org[[path]]" target="_blank">Run in your
+          browser on w3c-test.org</a></li>
+      </ul>
+    </div>
```

Comment:
This is nearly identical to: https://github.com/w3c/wptdashboard/blob/d7354d5a4c3735121cb37c845217039a3bdcb1f5/webapp/components/wpt-results.html#L117-L122

Reply:
Noted; Will split into a component in separate PR.